test/extended/prometheus/prometheus.go

```diff
@@ -49,6 +53,120 @@ type TelemeterClientConfig struct {
 	Enabled *bool `json:"enabled"`
 }
 
+var componentNamespace = os.Getenv("COMPONENT_NAMESPACE")
```

Comment:
`COMPONENT_NAMESPACE` is generic and and risks colliding with other uses elsewhere.
Is the test that slow? I'd prefer to do without it to simplify the logic.

Reply:
`COMPONENT_NAMESPACE` is not set in CI.

It is convenient for me, as a developer for one component, to use e.g., `COMPONENT_NAMESPACE=openshift-cluster-version`, when I am trying to figure out what is going on with my namespace, which has only ONE target. The debugging is much easier that way.

Yes, the test is slow if it go through all targets. If you do not set it up, i should just do all of them.

Reply:
`COMPONENT_NAMESPACE` is still generic, I'd go with sth like `MONITORING_AUTH_TEST_NAMESPACE` to help avoid any collision.

(I still think that during debugging one can always change the code to make it only consider some namespaces, I prefer to keep the logic simple, but let's see what others think about it)

Reply:
also, it'd be great to have a comment for this